### DISCUSSION TITLE
Answer Inline Query  How to add InlineKeyboardButton

### DISCUSSION THREAD

**Discussion by YC946586** [eternnoir/pyTelegramBotAPI#1796] [Q&A]
markup = types.InlineKeyboardMarkup()
 markup.add(types.InlineKeyboardButton("🌐testButton", url="http://www.google.com"))
 r = types.InlineQueryResultArticle('1', 'query',
 types.InputTextMessageContent(msg,parse_mode="Markdown",reply_markup=markup))
bot.answer_inline_query(query.id, [r])       

。。。reply_markup  
How should I send it? The current code is wrong

**Comment by Badiboy** [comment]
You cannot send any buttons for inline queriy answers.
